packages/cpp-debug/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "description": "Theia - C/C++ Debug Extension",
   "dependencies": {
-    "@theia/core": "next",
-    "@theia/debug": "next",
+    "@theia/core": "1.8.0",
+    "@theia/debug": "1.8.0",
```

Comment:
I think these 2 lines should be necessary since [we are already forcing the resolution of `@theia/core` and `@theia/debug` to `1.8.0`](https://github.com/ARMmbed/mbs-ide/blob/master/package.json#L87).

Unfortunately what happens if I don't explicitly change the Theia version is that a line is correctly added to the root `package.json` file pointing the `next` version to `1.8.0`, but the Theia dependencies are not found by the code in the `@theia/cpp-debug` package.

I welcome any suggestion of what might be going wrong.